### PR TITLE
added c-type casts

### PR DESCRIPTION
**Pull request by miyako**

### CODE REVIEW

**Comment by guillaume-kotulski** [comment]
Why including windows headers ?

**Comment by miyako** [comment]
that was my mistake

File: 4D Plugin API/4DPluginAPI.c [hunk 1]
```diff
@@ -22,6 +22,10 @@
 #include <stdlib.h>
 #include <string.h>
 
+#if __APPLE__
+#include <CoreGraphics/CoreGraphics.h>
```
Comment (guillaume-kotulski) [inline 4D Plugin API/4DPluginAPI.c:26]: Why is this include necessary?

Comment (miyako) [reply 4D Plugin API/4DPluginAPI.c:26]: in one of my projects I got an error where ``CGContextRef`` used in ``PA_UseQuartzAxis`` and ``PA_UseQuickdrawAxis `` was undefined.

Comment (guillaume-kotulski) [reply 4D Plugin API/4DPluginAPI.c:26]: Ok, prefer using `VERSIONMAC` which is defined in Flags.h than `__APPLE__`

**Pull request merged**